pkg/apis/storageos/v1/storageoscluster_types.go

```diff
@@ -308,12 +308,17 @@ func (s StorageOSClusterSpec) GetCSIClusterDriverRegistrarImage() string {
 }
 
 // GetCSIExternalProvisionerImage returns CSI external provisioner container image.
-func (s StorageOSClusterSpec) GetCSIExternalProvisionerImage(csiv1 bool) string {
+func (s StorageOSClusterSpec) GetCSIExternalProvisionerImage(csiv1 bool, v2 bool) string {
```

Comment:
`nodev2` would be a better name? Since we have a lot of images with different versions here, I feel it'd be better to change the name.